src/main/java/org/aleksey/springnew/dto/TodoUpdateDto.java

```diff
@@ -29,4 +30,30 @@ public class TodoUpdateDto {
     private String priority;
 
     private StatusType status;
+
+    public boolean equals(Object object) {
```

Comment:
You can use `@EqualsAndHashCode` Lombok annotation instead. And if you are overriding `equals` method, `hashCode` MUST BE overridden as well